subjects/ai/credit-scoring/README.md

```diff
@@ -1,38 +1,53 @@
 ## Credit scoring
 
+### Overview
+
 The goal of this project is to implement a scoring model based on various source of data ([check data documentation](./readme_data.md)) that returns the probability of default. In a nutshell, credit scoring represents an evaluation of how well the bank's customer can pay and is willing to pay off debt. It is also required that you provide an explanation of the score. For example, your model returns that the probability that one client doesn't pay back the loan is very high (90%). The reason behind is that variable_xxx which represents the ability to pay back the past loan is low. The output interpretability will appear in a visualization.
 
-The ability to understand the underlying factors of credit scoring is important. Credit scoring is subject to more and more regulation, so transparency is key. And more generally, more and more companies prefer transparency to black box models.
+### Role play
 
-### Resources
+Hey there, future credit scoring expert! Ready to dive into the exciting world of predicting loan defaults? You're in for a treat! This project is all about building a nifty model that can help figure out how likely someone is to pay back their loan. Cool, right?
+
+### Learning Objective
+
+The ability to understand the underlying factors of credit scoring is important. Credit scoring is subject to more and more regulation, so transparency is key. And more generally, more and more companies prefer transparency to black box models.
 
 Historical timeline of machine learning techniques applied to credit scoring
 
 - [Machine Learning or Econometrics for Credit Scoring: Let’s Get the Best of Both Worlds](https://hal.archives-ouvertes.fr/hal-02507499v3/document)
 
-### Scoring model
+### Instructions
+
+#### Scoring model
 
 There are 3 expected deliverables associated with the scoring model:
 
 - An exploratory data analysis notebook that describes the insights you find out in the data set.
 - The trained machine learning model with the features engineering pipeline:
 
   - Do not forget: **Coming up with features is difficult, time-consuming, requires expert knowledge. ‘Applied machine learning’ is basically feature engineering.**
-  - The model is validated if the **AUC on the test set is higher than 75%**.
+  - The model is validated if the **AUC on the test set is higher than 50%**.
```

Comment:
It was updated to be between 55% to 62%, the capacity of the devices was taken into consideration.

Reply:
it must be one specific value and not a range it can be more then 55% no need to specify 62% , and i really don't understand what is the relation between that and the devices capacity! as i know it about the model and how it trained